endpoints.json

```diff
@@ -9,6 +9,17 @@
       "topics": [{ "slug": "football", "description": "Footie!" }]
     }
   },
+  "GET /api/articles/:article_id": {
+    "description": "serves an article object of the requested article id",
+    "queries": [],
+    "exampleResponse": {
+      "article": 
+      {
+        "article_id":1,"title":"Living in the     shadow of a great man","topic":"mitch","author":"butter_bridge","body":"I find this existence challenging","created_at":"2020-07-09T20:11:00.000Z","votes":100,"article_img_url":"https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        "comment_count":"11"}
+      
```

Comment:
I would recommend formatting this so it is easier to read and tell what the keys actually are